tiktoken-rs/src/tokenizer.rs

```diff
@@ -30,6 +30,7 @@ pub enum Tokenizer {
 
 const MODEL_PREFIX_TO_TOKENIZER: &[(&str, Tokenizer)] = &[
     // chat
+    ("chatgpt-4o-latest", Tokenizer::O200kBase),
```

Comment:
Since this is a specific model and not a model prefix. Let's add it to `MODEL_TO_TOKENIZER`.